templates/non-ai-lab-chatbot/content/Containerfile

```diff
@@ -0,0 +1,10 @@
+FROM registry.access.redhat.com/ubi9/python-311:1-62.1716478620
+WORKDIR /app
+COPY requirements.txt .
+RUN pip install --upgrade pip
+RUN pip install --no-cache-dir --upgrade -r /app/requirements.txt
+COPY src/sample-app.py .
+ENV GRADIO_SERVER_NAME="0.0.0.0"
+EXPOSE 8501
+USER root
```

Comment:
future TODO: figure out how to run as non root

Reply:
I may have left that in by accident, let me do some testing to see if I can get that to run without `root`